package-lock.json

```diff
@@ -11,41 +11,41 @@
       "dev": true
     },
```

Comment:
There seems to be something wrong with either the package.json or package-lock.json in this PR.

When I locally run `npm ci` or `npm i` I get the following error:

````
➜  web.dev git:(design-tokens) npm i                                             
npm ERR! code EINTEGRITY
npm ERR! sha512-0Z+KTUitY4v1vSo19PsubSH0+mSjQPg73SmNcBOoICnvKbWecLgqwaDkZSyrbiMyxz6uwb/d8CL4hTm8kHn2zg== integrity checksum failed when using sha512: wanted sha512-0Z+KTUitY4v1vSo19PsubSH0+mSjQPg73SmNcBOoICnvKbWecLgqwaDkZSyrbiMyxz6uwb/d8CL4hTm8kHn2zg== but got sha512-Tonwb/arx6j42KQfA4ym+Q/IxWY1svLl8AXnzqLXW/DaQ/z8N1qAV4T9Nbl+Pb0ilvQy+5QNl7+Cu3RjRASYyg==. (10338 bytes)
```

I'm on Node v14.14.0, npm 6.14.8

Reply:
ok well that's weird. I checked out main, installed things, then checked out this branch again, and ran `npm ci` and everything worked. So maybe the issue was just on my machine. That would explain why the Actions weren't having install errors. So i think we can ignore this but I'll leave the note in case anyone else hits a similar issue.